data/data/nutanix/cluster/main.tf

```diff
@@ -29,7 +29,7 @@ resource "nutanix_category_value" "ocp_category_value_shared" {
 
 resource "nutanix_image" "rhcos" {
   name        = var.nutanix_image
-  source_path = var.nutanix_image_filepath
+  source_url  = var.nutanix_image_url
```

Comment:
I think its source_uri

Reply:
Correct, I just created this PR to give you an idea about the scope of the change :)